org.kde.CrowTranslate.yaml

```diff
@@ -0,0 +1,87 @@
+id: org.kde.CrowTranslate
+sdk: org.kde.Sdk
+runtime: org.kde.Platform
+runtime-version: 5.15-23.08
+add-extensions:
+  org.kde.CrowTranslate.tessdata:
+    directory: share/tessdata
+    version: stable
+    versions: stable;beta;test
+    no-autodownload: true
+    autodelete: true
+finish-args:
+  - --device=dri
+  - --env=TESSDATA_PREFIX=/app/share
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+copy-icon: true
+command: crow
+separate-locales: false
+cleanup:
+  - '*.a'
+  - '*.la'
+  - /include
+  - /lib/cmake
+  - /lib/pkgconfig
+modules:
+  - name: crow-translate
+    buildsystem: cmake-ninja
+    sources:
+      - type: git
+        url: https://invent.kde.org/office/crow-translate.git
+        # tag: 3.0.0 Not tagged yet
+        commit: 4a31b516292455e9e905a905f69101865fddb823
+        x-checker-data:
+          type: git
+    modules:
+      - name: tesseract
+        buildsystem: cmake-ninja
+        builddir: true
+        config-opts:
+          - -DBUILD_SHARED_LIBS:BOOL=ON
+        cleanup:
+          - /bin
+        sources:
+          - type: archive
+            url: https://github.com/tesseract-ocr/tesseract/archive/refs/tags/5.4.1.tar.gz
+            sha256: c4bc2a81c12a472f445b7c2fb4705a08bd643ef467f51ec84f0e148bd368051b
+            x-checker-data:
+              type: anitya
+              project-id: 4954
+              url-template: https://github.com/tesseract-ocr/tesseract/archive/refs/tags/$version.tar.gz
+        modules:
+          - name: leptonica
+            buildsystem: cmake-ninja
+            builddir: true
+            config-opts:
+              - -DBUILD_SHARED_LIBS:BOOL=ON
+            sources:
+              - type: archive
+                url: https://github.com/DanBloomberg/leptonica/releases/download/1.84.1/leptonica-1.84.1.tar.gz
+                sha256: 2b3e1254b1cca381e77c819b59ca99774ff43530209b9aeb511e1d46588a64f6
+                x-checker-data:
+                  type: anitya
+                  project-id: 1549
+                  url-template: https://github.com/DanBloomberg/leptonica/releases/download/$version/leptonica-$version.tar.gz
+
+      # The `tessdata_fast` variant is recommended for end users: https://tesseract-ocr.github.io/tessdoc/Data-Files
+      - name: tessdata_fast
+        buildsystem: simple
+        build-commands:
+          - rm -rf ${FLATPAK_DEST}/share/tessdata
+          - mkdir -p ${FLATPAK_DEST}/share/tessdata
+          - cp -av * ${FLATPAK_DEST}/share/tessdata/
+        sources:
+          - type: git
+            url: https://github.com/tesseract-ocr/tessdata_fast
```

Comment:
Can this be fetched from a tarball instead of Git?

Reply:
Fixed, thanks!